packages/databricks-vscode/src/bundle/bundleAutocompleteProvider.ts

```diff
@@ -28,14 +28,21 @@ export async function registerBundleAutocompleteProvider(
         context.subscriptions.push(
             bundleWatcher.onDidChangeRootFile(async () => {
                 bundleFileList = await bundleFileSet.allFiles();
+            }),
+            bundleWatcher.onDidCreate(async (e) => {
+                bundleFileList.push(e);
+            }),
+            bundleWatcher.onDidDelete(async (e) => {
+                bundleFileList.push(e);
```

Comment:
Shouldn't we remove the file from this list on delete?

I might be misunderstanding how this works.